### ISSUE TITLE
About multivariate time series

### ISSUE THREAD

**Issue by hojie11** [ML4ITS/TimeVQVAE-AnomalyDetection#3]
Thanks for your nice work. It is really helpful to do anomaly detection on univariate time series.
I think your method could be applied to multivariate time series data. have you tried on the multivariate?

**Comment by danelee2601** [comment]
of course, that's a natural transition. 
Short answer: not yet.
Long answer: when we just started working on TSAD, we noticed that there had been a lot of progress illusion due to wrong metrics, as described in our paper. Therefore, we first wanted to build a robust TSAD model for univariate time series, and then move on to multivariate time series because we believe in incremental improvement. Despite the availability of several obvious ways to modify TimeVQVAE-AD for multivariate time series, I don't think our team will be working on it in the near future because of the end of funding project and my PhD program. 

But I'm sure it'd be a good research topic with highly promising feasibility.
